### DISCUSSION TITLE
如何改变坐标轴的原点

### DISCUSSION THREAD

**Discussion by heiansenlin** [jfree/jfreechart#306] [Q&A]
如题
在使用过程中，Y轴与Y轴的交汇的原点总是在左下角，而我希望能够将X轴放置在Y轴的中间位置，或者将Y轴放置到X轴的中间位置
而不是一直在图表的边框上
请问有什么好的解决方案吗？

**Comment by trashgod** [comment]
This [example](https://stackoverflow.com/a/20359200/230513) illustrates using crosshairs and adjusting axis range.
